smf/chrony-setup/manifest.xml

```diff
@@ -12,21 +12,21 @@
   </dependency>
 
   <exec_method type='method' name='start'
-    exec='/opt/oxide/zone-setup-cli/bin/zone-setup chrony-setup -b %{config/boundary} -s %{config/server} -a %{config/allow}'
+    exec='/opt/oxide/zone-setup-cli/bin/zone-setup chrony-setup -b %{config/boundary} -p %{config/boundary_pool} -s %{config/server} -a %{config/allow}'
     timeout_seconds='0'>
   <method_context security_flags="aslr">
   <method_credential user="root" group="root"
     privileges="basic,file_chown" />
   </method_context>
   </exec_method>
 
-  
   <property_group name='startd' type='framework'>
     <propval name='duration' type='astring' value='transient' />
   </property_group>
 
   <property_group name="config" type="application">
     <propval name="boundary" type="boolean" value="false" />
+    <propval name="boundary_pool" type="astring" value="" />
```

Comment:
Are these properties documented anywhere?

Reply:
Only the `zone-setup` help output. I added comments in the manifest.